[CS2113T-T09-3] MealCompanion

Jjzeng123 opened the pull request:
Recipe and Ingredient tracker for SoC students staying in halls, optimized for a CLI interface

ZhongXiangWong commented on docs/UML diagrams/AddIngredientSequence.puml:
```diff
@@ -0,0 +1,38 @@
+
+":AddCommand" -> ":IngredientList": getIngredients()
+activate ":IngredientList"
+":AddCommand" <- ":IngredientList": ArrayList<Ingredient>: ingredients
```
return line should be dashed instead of solid.

ZhongXiangWong commented on docs/UML diagrams/AddIngredientSequence.puml:
```diff
@@ -0,0 +1,38 @@
+":AddCommand" -> ":IngredientList": getIngredients()
+activate ":IngredientList"
+":AddCommand" <- ":IngredientList": ArrayList<Ingredient>: ingredients
+deactivate ":IngredientList"
+
+":AddCommand" -> ":AddCommand": findIndex("egg")
+activate ":AddCommand"
+
+":AddCommand" -> ":AddCommand": return -1
+
+deactivate ":AddCommand"
+
```
Can use an X at the end of the lifeline of an object to show its deletion for AddCommand.

JangusRoundstone replied:
Maybe you can use an X at the end of lifeline to show object deletion

wenxin-c commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+3. Open the command terminal and navigate to the folder where you downloaded `mealcompanion.jar` using `cd <file path>` commands as necessary.
+4. Ensure the terminal is within the correct folder.
+5. Use the command `java -jar mealcompanion.jar` to launch Meal Companion.
+![startup1.png](images/startup1.png)
```
This picture doesn't seem to be rendered on the developer guide, you might want to check whether there is error in the path?

wenxin-c commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+the command in such a way that we do not have to worry about match priority or ordering, as we would if we
+were to implement this with simple if-else or switch statements.
+
+![RouterUML.png](images/RouterUML.png)
```
I think the `CommandRouterNode` class should be connected to its interface `Routable` using an dotted arrow instead of solid arrow. Maybe you can double check on this?

wenxin-c replied:
For UML methods, the return type should be at the back of the method instead of at the back. For example, `getRouteName(): String`. Maybe you can check it out?

nicholas132000 replied:
![image](https://user-images.githubusercontent.com/88270205/228759525-4a4c2417-c4ef-41cb-b0a7-3288ebfdf98d.png)

Perhaps you could make the uml diagram more aligned to the one in lecture notes by adding these 2 lines on the top of your source file
`skinparam classAttributeIconSize 0`
`hide circle`

nicholas132000 replied:
![image](https://user-images.githubusercontent.com/88270205/228762560-226c08b4-c225-4312-b3a2-10742cbc53e3.png)
I think you should use a dotted arrow to show that CommandRouterNode class implements the Routable interface

JangusRoundstone replied:
<img width="991" alt="Screen Shot 2023-03-31 at 10 46 27 AM" src="https://user-images.githubusercontent.com/88696064/229010073-127977b7-db5e-4ef4-8ea7-c24d2672f8fb.png">

I believe the "implement" function should be represented by a dashed arrow with a solid head.

keanecxz commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+the command in such a way that we do not have to worry about match priority or ordering, as we would if we
+were to implement this with simple if-else or switch statements.
+
+![RouterUML.png](images/RouterUML.png)
+
```
In this diagram, is the dotted line connecting CommandRouternode and ExecutableCommandFactory missing a line?
![image](https://user-images.githubusercontent.com/108349014/228732698-8dd6d305-d8c9-42e8-a1c3-b56f0e3a94ac.png)

wenxin-c commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+
+The sequence diagram below illustrates the process for resolving the "recipe all" command.
+
+![RouterSequenceUML.png](images/RouterSequenceUML.png)
```
Maybe the activation bar for `root:CommandRouterNode` should end after the boxes instead of before it (i.e. activation ends after finishing the most outer loop)?

keanecxz commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+This is an object diagram representing a portion of the routing tree for MealCompanion, supporting the "recipe all",
+"recipe possible", and "ingredient list" commands.
+
+The sequence diagram below illustrates the process for resolving the "recipe all" command.
+
+![RouterSequenceUML.png](images/RouterSequenceUML.png)
+
```
In this sequence diagram, should the activation bar be disconnected (in rootChildren, recipe children and RecipeAllCommandFactory... is new instance of the class is being created?
![image](https://user-images.githubusercontent.com/108349014/228732603-8e74f168-f119-49d4-9437-3ff10f435b2d.png)

wenxin-c commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+
+## Implementation
+
+### Add and Remove Command
```
Very neat and systematic way in explaining the add and remove command

keanecxz commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+
+### Ingredient Class
+
+Below shows the class diagram of how ingredients are being stored in our program
+
+![IngredientUML.png](images/IngredientUML.png)
```
In this class diagram, could the number of IngredientList in MealCompanionSession be fixed to 1?
![image](https://user-images.githubusercontent.com/108349014/228732211-84982a03-2276-42c7-9f7a-6fef8145da22.png)

keanecxz replied:
In this class diagram, should the red squares and solid green circles be minuses and plusses instead?
![image](https://user-images.githubusercontent.com/108349014/228732145-f6f9a4b2-3de7-452f-b41a-74e7849ce3e3.png)

matthew-liu-zhenjie replied:
![Screenshot from 2023-03-31 13-15-51](https://user-images.githubusercontent.com/2891116/229029364-b6a2950b-a1c2-4a44-9768-85d59a6e595f.png)

There should not be a class circle in the UML diagram. This can be removed with `hide circle`. There should also not be red rectangles and green circles but rather minuses and plusses to indicate visibility.

wenxin-c commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+
+### User Stories
+
+#### v1.0
```
It is good to separate V1.0 and V2.0 user stories.

matthew-liu-zhenjie replied:
It was helpful that V1.0 and V2.0 user stories were written in two separate tables.

wenxin-c reviewed (commented):
Overall very neat and systematic developer guide! UML and sequence diagrams are clear as well. But there are some implementations you might wish to double check. Good work!

keanecxz reviewed (commented):
Overall, neat and concise, perhaps some changes will make it better

nicholas132000 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+
+## Appendix: Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
```
Perhaps you could consider adding more details to this part of the developer guide to make it more comprehensive

nicholas132000 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+
+Below shows the sequence diagram for the above AddCommand:
+
+![AddIngredientSequenceUML.png](images/AddIngredientSequenceUML.png)
```
![image](https://user-images.githubusercontent.com/88270205/228765092-1c39aecc-1fd2-499f-9b00-92f0a5eadfa7.png)
Does the return -1 of the findIndex("egg) self-invocated method indicate that the egg is not found? Perhaps you could be clearer. Also, return should be represented with a dotted line

lihka1202 commented on docs/AboutUs.md:
```diff
@@ -1,9 +1,10 @@
-![](https://via.placeholder.com/100.png?text=Photo) | Ron John | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | John Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Don Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
+ Display                                             |         Name         |               Github Profile                |               Portfolio               
```
Great idea to abstract out the "About Us" section from the leading Developers Guide! Do note that the photos still need to render on the web page!

lihka1202 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
 # Developer Guide
+Welcome to the Meal Companion Developer Guide! Thank you for taking an interest in the behind-the-scenes working of our product, and we hope this document proves informative and useful for your work.
+
+## Table of Contents
```
I like how hyperlinks have been used for easier access!

lihka1202 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+
+![RemoveIngredientSequenceUML.png](images/RemoveIngredientSequenceUML.png)
+
+###### [Back to table of contents](#table-of-contents)
```
I like the usage of "Back to the table of contents" so that readers can find their way back to the "Table of contents" with ease!

lihka1202 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
-## Design & implementation
+###### [Back to table of contents](#table-of-contents)
+
+## Setting up, getting started
```
Very clear instructions, however there is generally no need to use the terminal in Intellij to test the functionality of the program!

lihka1202 reviewed (commented):
Great Job on the Overall!

JangusRoundstone commented on docs/UML diagrams/Ingredients.puml:
```diff
@@ -0,0 +1,32 @@
+@startuml
+'https://plantuml.com/class-diagram
+
+IngredientList <-- MealCompanionSession
+Ingredient "*" <-- IngredientList
+
+class Ingredient {
+-quantity: double
+-name: String
++getQuantity()
++setQuantity(double)
++getName()
+}
+
+class IngredientList {
+
++size(): int
++remove(int)
++get(int): Ingredient
++getIngredients() : IngredientList
+}
+
+Class MealCompanionSession {
+- controlFLow: MealCompanionControlFlow
+- recipes: RecipeList
+- ui: MealCompanionUI
+- ingredientStorage: IngredientStorage
+
+}
+
+
```
<img width="825" alt="Screen Shot 2023-03-31 at 11 26 08 AM" src="https://user-images.githubusercontent.com/88696064/229015255-32d0c9d8-908f-476d-929d-b6636d6f22ba.png">

The "+" and "-" are not showing correctly here.

JangusRoundstone commented on README.md:
```diff
@@ -1,28 +1,19 @@
 ## Setting up in Intellij
 
 Prerequisites: JDK 11 (use the exact version), update Intellij to the most recent version.
 
 1. **Ensure Intellij JDK 11 is defined as an SDK**, as described [here](https://www.jetbrains.com/help/idea/sdk.html#set-up-jdk) -- this step is not needed if you have used JDK 11 in a previous Intellij project.
 1. **Import the project _as a Gradle project_**, as described [here](https://se-education.org/guides/tutorials/intellijImportGradleProject.html).
-1. **Verify the set up**: After the importing is complete, locate the `src/main/java/seedu/duke/Duke.java` file, right-click it, and choose `Run Duke.main()`. If the setup is correct, you should see something like the below:
+1. **Verify the set up**: After the importing is complete, locate the `src/main/java/seedu/mealcompanion/MealCompanion.java` file, right-click it, and choose `Run MealCompanion.main()`. If the setup is correct, you should see something like the below:
```
Comprehensive guide for complete new users to the system. Good!

JangusRoundstone reviewed (commented):
Overall looks good! Good job!

matthew-liu-zhenjie commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+
+The following sequence diagram shows how the Recipe Detail Command works:
+
+![RecipeDetailCommandSequence.png](images%2FRecipeDetailCommandSequence.png)
```
![Screenshot from 2023-03-31 13-18-58](https://user-images.githubusercontent.com/2891116/229029958-ce523bb6-d22f-4481-8e2b-bc1248e3f37f.png)

The activation arrow for getingredients() is directed to the recipe instead of recipes:RecipeList.

:RecipeDetailCommand is also not deactivated at the end.

matthew-liu-zhenjie commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+#### Value proposition
+Our product targets students who would like to save money on buying or ordering food, and also to save time by preparing their meals beforehand, so they can eat immediately after a class without having to queue for food or look for a seat in a crowded canteen on campus. Aside from RCs, hostels on campus do not provide meal plans, so our product addresses this by helping users pick up cooking and  ...
+
+###### [Back to table of contents](#table-of-contents)
```
This is a very handy use of hyperlinks to improve navigability of the DG.

matthew-liu-zhenjie commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,275 @@
+
+## Setting up, getting started
+
+1. Java 11 is required for Meal Companion to function properly, please make sure your device has Java 11 installed. If not, you may download it [here](https://www.oracle.com/sg/java/technologies/downloads/#java11). If you need further asssistance installing Java 11, you may refer to the [Installation Guide](https://docs.oracle.com/en/java/javase/11/install/installation-jdk-microsoft-windows-platf ...
```
It was good to see that the version of Java to be used was specified here.

Jjzeng123 committed:
Merge branch 'master' into PPP

Jjzeng123 committed:
Merge pull request #152 from jingyaaa/PPP

PPP Draft

Jjzeng123 committed:
Rename Jjzeng123.md to jjzeng123.md

Jjzeng123 committed:
Update AboutUs.md

Commit:
Update print error messages using UI class

Commit:
Fix recipe need and recipe almost bugs

Commit:
Fix recipe find bug and add test

Commit:
Fix checkstyle

jingyaaa committed:
Merge pull request #144 from TJW0911/Bug-fixes

Set maximum ingredient amount to 10000 and reduced method chaining

EthanYidong committed:
Add extended value proposition to front matter

EthanYidong committed:
Update EthanYidong's PPP

ngyida committed:
Merge pull request #155 from EthanYidong/EthanYidong/ppp

Update EthanYidong's PPP

ngyida committed:
make recipe detail non case sensitive

ngyida committed:
fix error message for negative index in recipedetail

ngyida committed:
Update puml diagrams

ngyida committed:
Update PPP

TJW0911 committed:
Merge pull request #154 from jingyaaa/FixPEDBugs

Fix ped bugs

TJW0911 committed:
Merge pull request #156 from ngyida/wk13

Minor bug fixes + DG update

EthanYidong committed:
Overhaul error handling

We need to standardize error handling
across all commands

Without standardized error handling, a lot of code is repeated
and it becomes hard to locate potential bugs with parsing

Let's,
* implement "extractors" which act as gates to provide formatted
values from arguments (strings, integers, recipe, ingredient)
* update commands to accept the already-validated forms of
arguments
* update tests to use the new command format

Also,
* refractor factory classes into corresponding packages

aaronxujiachen committed:
Added logo for Meal Companion

Commit:
Add architecture diagram

EthanYidong committed:
Implement printing command usage on misformed input

TJW0911 committed:
Updated DG

EthanYidong committed:
Add documentation for extractor classes

TJW0911 committed:
Updated DG

aaronxujiachen committed:
Adjusted the resolution of Meal Companion logo

TJW0911 committed:
Update tjw0911.md

TJW0911 committed:
Update tjw0911.md

TJW0911 committed:
Update tjw0911.md

aaronxujiachen committed:
Added farewell message and image

aaronxujiachen committed:
Fix checkstyle error

EthanYidong committed:
Write tests for extractors

aaronxujiachen committed:
Modified the code of logo

EthanYidong committed:
Merge commit '1e8d3df3a160a90ccf7a7beeeee6c9b7fcaa46a4' into EthanYidong/Error-Handling

# Conflicts:
#	src/main/java/seedu/mealcompanion/MealCompanionSession.java
#	src/main/java/seedu/mealcompanion/command/recipe/RecipeDetailCommand.java
#	src/main/java/seedu/mealcompanion/command/recipe/RecipeNeedCommand.java
#	src/main/java/seedu/mealcompanion/recipe/RecipeList.java

EthanYidong committed:
Fix regression with `hello world` command
Fix bug introduced with introduction logos

EthanYidong committed:
Merge commit 'b1496d918eecc8d715b99c0cb2c916f4a0204d62' into EthanYidong/Error-Handling

# Conflicts:
#	src/main/java/seedu/mealcompanion/ui/MealCompanionUI.java

Commit:
Update testing portion

Commit:
Add activity diagram

Commit:
Update PPP

ngyida committed:
Merge pull request #158 from TJW0911/DG

Updated DG

ngyida committed:
Merge pull request #159 from EthanYidong/EthanYidong/Error-Handling

Standardize error handling

Jjzeng123 committed:
Merge pull request #157 from jingyaaa/Update_Developer_Guide

Update DG

aaronxujiachen committed:
Updated PPP for Xu JiaChen

Jjzeng123 committed:
Merge branch 'master' of https://github.com/Jjzeng123/tp

aaronxujiachen committed:
Merge pull request #160 from aaronxujiachen/master

Updated PPP for Xu JiaChen

ngyida committed:
Update UG

ngyida committed:
Update PPP

Jjzeng123 committed:
Merge branch 'master' of https://github.com/Jjzeng123/tp

Jjzeng123 committed:
Merge pull request #161 from ngyida/yiDa-wk13

Update UG

Jjzeng123 committed:
Merge branch 'master' of https://github.com/Jjzeng123/tp

EthanYidong committed:
Fix error message for IntegerExtractor

EthanYidong committed:
Fix case sensitivity of ingredient parsing

EthanYidong committed:
Fix regression on Ingredient search command

Jjzeng123 committed:
Update DeveloperGuide.md

Jjzeng123 committed:
Merge pull request #162 from EthanYidong/EthanYidong/Error-Handling

Fix regressions caused by error handling standardization

Jjzeng123 committed:
Merge branch 'master' of https://github.com/Jjzeng123/tp

TJW0911 committed:
Updated help messages

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Merge pull request #163 from TJW0911/Help-Messages

Updated help messages

Commit:
Fix storage bugs

Commit:
Fix PPP line break

jingyaaa committed:
Merge pull request #166 from jingyaaa/Update_PPP

Update PPP

Commit:
Minor fix to PPP

jingyaaa committed:
Merge pull request #167 from jingyaaa/Update_PPP

Minor fix to PPP

TJW0911 committed:
Updated PPP

TJW0911 committed:
Merge branch 'master' into master

TJW0911 committed:
Merge pull request #168 from TJW0911/master

Updated PPP

TJW0911 committed:
Merge pull request #164 from Jjzeng123/master

Add malformed input warnings and file corruption warning and tip to UG

TJW0911 committed:
Merge pull request #165 from jingyaaa/FixStorage

Fix storage bugs

TJW0911 committed:
Changed name in PPP to own full name

TJW0911 committed:
Merge branch 'master' of https://github.com/TJW0911/tp

TJW0911 committed:
Merge pull request #169 from TJW0911/master

Changed name in PPP to own full name

TJW0911 committed:
Added hyperlinks to PPP

TJW0911 committed:
Added hyperlinks

TJW0911 committed:
Added hyperlink

TJW0911 committed:
Merge pull request #170 from TJW0911/master

Added hyperlinks to PPP

EthanYidong committed:
Add code quality fixes

Jjzeng123 committed:
Update jjzeng123.md

Jjzeng123 committed:
Update jjzeng123.md

Commit:
Add recipe features to DG

Jjzeng123 committed:
Merge pull request #172 from Jjzeng123/master

Update PPP formatting

Jjzeng123 committed:
Merge pull request #171 from EthanYidong/EthanYidong/Fix-Code-Quality

Fix Code Quality Issues

Commit:
Fix link

Commit:
Fix images

Jjzeng123 committed:
Merge branch 'master' of https://github.com/Jjzeng123/tp

Commit:
Add links

Jjzeng123 committed:
Delete startup.png

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Delete recipepossible1.png

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Delete recipepossibleallergen.png

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Delete make.png

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Delete add1.png

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Delete remove1.png

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Delete ingredientlist1.png

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Delete clear.png

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Delete recipedetails2.png

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Delete recipeneed.png

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Delete recipealmost.png

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Update UserGuide.md

TJW0911 committed:
Merge pull request #173 from jingyaaa/DG_updates

Update DG

Jjzeng123 committed:
Update jjzeng123.md

Jjzeng123 committed:
Merge pull request #174 from Jjzeng123/master

Update UG with Recipe find command and screenshots for double -> int and added hyperlinks within each sub-section to navigate back to the large section

Jjzeng123 committed:
Merge branch 'master' of https://github.com/Jjzeng123/tp

TJW0911 committed:
Updated exception message

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Merge pull request #175 from TJW0911/master

Updated exception message

Jjzeng123 committed:
Merge pull request #176 from Jjzeng123/master

Add mini-tables of contents for each section

EthanYidong committed:
Add logging to MealCompanionSession

EthanYidong committed:
Remove magic literal

TJW0911 committed:
Merge pull request #177 from EthanYidong/EthanYidong/Improve-Code-Quality

Improve code quality

Jjzeng123 committed:
Update _config.yml

Jjzeng123 committed:
Update _config.yml

EthanYidong committed:
Add extractor information to the developer guide

Jjzeng123 committed:
Update _config.yml

Jjzeng123 committed:
Update _config.yml

Jjzeng123 committed:
Update _config.yml

aaronxujiachen committed:
Fixed empty ingredients list issue

Jjzeng123 committed:
Update _config.yml

Jjzeng123 committed:
Merge pull request #178 from EthanYidong/EthanYidong/Final-Doc-Update

Add extractor information to the developer guide

TJW0911 committed:
Merge pull request #179 from aaronxujiachen/master

Fixed empty ingredients list issue

Jjzeng123 committed:
Merge branch 'master' of https://github.com/Jjzeng123/tp

EthanYidong committed:
Update EthanYidong's PPP
Update architecture diagram and escription in the DG

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Delete reciperandom.png

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update README.md

Jjzeng123 committed:
Update README.md

Jjzeng123 committed:
Update README.md

ngyida committed:
update DG and add unit test

EthanYidong committed:
Update EthanYidong's PPP
Add exception handling to DG

EthanYidong committed:
Update Router UML diagram

ngyida committed:
Merge branch 'master' into yiDa-wk13

TJW0911 committed:
Merge pull request #180 from EthanYidong/EthanYidong/Final-Doc-Update

Update DG and Ethan's PPP

ngyida committed:
update DG

TJW0911 committed:
Merge pull request #181 from ngyida/yiDa-wk13

update DG and add unit test

EthanYidong committed:
Fix bug in DG

Jjzeng123 committed:
Merge branch 'master' of https://github.com/Jjzeng123/tp

ngyida committed:
Update DG

ngyida committed:
Merge pull request #182 from EthanYidong/EthanYidong/Final-Doc-Update

Fix bug in DG

Jjzeng123 committed:
New easter egg

Jjzeng123 committed:
Merge branch 'AY2223S2-CS2113T-T09-3:master' into master

TJW0911 committed:
Merge pull request #183 from Jjzeng123/master

new easter egg

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update DeveloperGuide.md

ngyida committed:
Update ngyida.md

EthanYidong committed:
Add acknowledgements to DG

EthanYidong committed:
Update Ethan's PPP

Jjzeng123 committed:
Delete jar2.png

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update DeveloperGuide.md

EthanYidong committed:
Merge pull request #184 from EthanYidong/EthanYidong/Final-Doc-Update

Add acknowledgements to DG

ngyida committed:
Update DeveloperGuide.md

Jjzeng123 committed:
Update DeveloperGuide.md

EthanYidong committed:
Update home page

EthanYidong committed:
Merge pull request #185 from EthanYidong/EthanYidong/Final-Doc-Update

Update home page

Jjzeng123 committed:
Update DeveloperGuide.md

Jjzeng123 committed:
Update jjzeng123.md

Jjzeng123 committed:
Merge pull request #186 from Jjzeng123/master

Update DG with manual testing

Jjzeng123 committed:
Update DeveloperGuide.md

TJW0911 committed:
Merge pull request #187 from Jjzeng123/master

Update DeveloperGuide.md

Jjzeng123 committed:
Update DeveloperGuide.md

Jjzeng123 committed:
Merge pull request #188 from Jjzeng123/master

Update DeveloperGuide.md

EthanYidong committed:
Fix table in DG

Jjzeng123 committed:
Merge pull request #189 from EthanYidong/EthanYidong/Final-Doc-Update

Fix table in DG

Jjzeng123 committed:
Update DeveloperGuide.md

Jjzeng123 committed:
Update DeveloperGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Merge pull request #190 from Jjzeng123/master

Testing if page breaks work correctly

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Merge pull request #191 from Jjzeng123/master

Test formatting changes

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Merge pull request #192 from Jjzeng123/master

Testing more changes to formatting for saving

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Merge pull request #193 from Jjzeng123/master

Update with CS2101 requirements

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Merge pull request #194 from Jjzeng123/master

More changes to update for CS2101

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

TJW0911 committed:
Updated ingredient commands to conform with a fixed format and added linking statements between sections

Jjzeng123 committed:
Merge pull request #195 from TJW0911/master

Updated ingredient commands to conform with a fixed format and added …

Jjzeng123 committed:
Update UserGuide.md

jingyaaa committed:
Update user guide 

- second half of recipes
- storage
- help

jingyaaa committed:
update UG

jingyaaa committed:
Merge pull request #196 from jingyaaa/master

Update UG

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Merge pull request #197 from Jjzeng123/master

Embed Video

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Merge pull request #198 from Jjzeng123/master

Add images for copy and paste files

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Merge pull request #199 from Jjzeng123/master

Resize images

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

aaronxujiachen committed:
Updated Allergen Features with spacing.

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Merge pull request #201 from Jjzeng123/master

Fix image resizing

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Merge pull request #202 from Jjzeng123/master

Cleanup some parts

aaronxujiachen committed:
Merge branch 'master' into master

aaronxujiachen committed:
Merge pull request #200 from aaronxujiachen/master

Updated Allergen Features with spacing.

ngyida committed:
update UG

ngyida committed:
Merge pull request #203 from ngyida/yiDa-UpdateUG

update UG

EthanYidong committed:
Update UG with Terminal and CLI instructions

EthanYidong committed:
Merge pull request #204 from EthanYidong/EthanYidong/Final-Doc-Update

Update UG with Terminal and CLI instructions

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Add files via upload

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

EthanYidong committed:
Center tagline

EthanYidong committed:
Merge pull request #205 from EthanYidong/EthanYidong/Final-Doc-Update

Center tagline

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md

Jjzeng123 committed:
Update UserGuide.md
